libs/application/template-api-modules/src/lib/modules/templates/new-primary-school/new-primary-school.utils.ts

```diff
@@ -106,14 +128,83 @@ export const transformApplicationToNewPrimarySchoolDTO = (
           }
         : {}),
     },
+    ...(applicationType === ApplicationType.ENROLLMENT_IN_PRIMARY_SCHOOL
+      ? {
+          health: {
+            ...(hasFoodAllergiesOrIntolerances?.includes(YES)
+              ? {
+                  foodAllergiesOrIntolerances: foodAllergiesOrIntolerances,
```

Comment:
nitpick. Use object property shorthand
```suggestion
                  foodAllergiesOrIntolerances,
```

Reply:
Fixed 💯